libs/designer/src/lib/core/state/panel/panelSlice.ts

```diff
@@ -150,13 +150,15 @@ export const panelSlice = createSlice({
     showDefaultTabs: (
       state,
       action: PayloadAction<
-        { isScopeNode?: boolean; isMonitoringView?: boolean; hasSchema?: boolean; showRunHistory?: boolean } | undefined
+        | { isScopeNode?: boolean; isMonitoringView?: boolean; isUnitTest?: boolean; hasSchema?: boolean; showRunHistory?: boolean }
```

Comment:
if we are just showing the unit test tab, consider using the isolateTab action

Reply:
Sure, I will take a look at it. Thanks!